### ISSUE TITLE
Page title

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#583]
Issue group: Misc
Visibility: 2
Evaluation: Partially Supports
Success Criteria: 2.4.2: Page Titled
Needs design: Content

### Description
Most pages have unique meaningful titles. However, there are some case where a modification or additional steps retain the page title of the parent or first step. This may cause some confusion/disorientation about the page change. Each page should have a unique title that describes the topic/purpose of the page.

For example: 
- [x] The Manage page for a Template folder has the same page title as viewing the Template folder. 
- [x] When sending a message, each personalization step has the same page title. 

![]()

**Comment by amazingphilippe** [comment]
Manage page for template folder has been fixed in https://github.com/cds-snc/notification-admin/pull/1244

**Comment by yaelberger-commits** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/583?planning-poker) with Zenhub @amazingphilippe

**Comment by yaelberger-commits** [comment]
Phil needs to fix some tests. Changes titles and tests are breaking, good sign...

**Comment by YedidaZalik** [comment]
PR is in for review, tests passing.

**Comment by YedidaZalik** [comment]
Ready for review when there's time.

**Comment by YedidaZalik** [comment]
@sastels reviewed and merged last week, deployed to prod yesterday
@andrewleith to QA
